vaadin-dev/pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>vaadin-dev-bundle</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.vaadin.hilla</groupId>
+            <artifactId>hilla-dev-mode</artifactId>
+        </dependency>
+
```

Comment:
why is this needed here instead of on the spring starter ?

Reply:
Discussed in slack. Conclusion: remove it from spring starter.